tests/e2e/api/README.md

```diff
@@ -51,12 +51,20 @@ httpClient.get( '/wc/v3/products' ).then( ( response ) => {
 
 As a convenience utility we've created repositories for core data types that can simplify interacting with the API:
 
+#### Parent/Base Repositories
```

Comment:
Eventually, I wonder if it would be helpful to split this out into a table that helps map these relationships, something like:

#### Base repositories
* Coupon
* Order
etc

### Parent / child repositories
| Parent repository | Child repository |
|------------------|------------------|
| Settings group    | Setting                 |
| Variable product | Product Variation |

Mostly just a thought that this could be useful as we add more in the future that follow this parent/child model.

Reply:
> Eventually, I wonder if it would be helpful to split this out into a table that helps map these relationships, something like:

This sounds good once we have more child repositories.